### DISCUSSION TITLE
Add conditional rendering of SIWE tooltip

### DISCUSSION THREAD

**Discussion by Drewsapple** [family/connectkit#304] [Ideas]
When SIWE is enabled, the profile page of the modal has a tooltip and a badge on the SIWE button in the top left that says "You're not signed in to this app. Sign in with Ethereum to continue." While this is helpful to have as a base case, I have some pages in my app that don't need SIWE, but just a connected wallet. I have the `disableSiweRedirect: true` flag enabled in my ConnectKit Provider options, which stops most users from feeling they need to sign-in (as the connect modal closes after connecting).

However, users going to disconnect/reconnect on these pages see this tooltip and get confused. I'd like to disable the tooltip (the badge is fine), as all of my SIWE-requiring pages have a separate button that requests the signature if they're not signed in.

IMO, the tooltip should disappear if with `disableSiweRedirect: true` but a separate flag for this would also work. 

**I'd be happy to implement this in a PR** if someone at Family gave me direction on how the option should be expressed in the provider.

**Comment by Drewsapple** [comment]
For my own use:
[Current conditional rendering location](https://github.com/family/connectkit/blob/825f95e8f6bd9fae16803a63efaa14b87137a793/packages/connectkit/src/components/Common/Modal/index.tsx#L512), I'd just need to add a condition to this. i.e.
the existing `{(!isSignedIn && !context.options?.disableSiweRedirect) (`
or a new `disableSiweTooltip` option `{(!isSignedIn && !context.options?.disableSiweTooltip) (`, adding the corresponding name to the [ContextValue type here](https://github.com/family/connectkit/blob/825f95e8f6bd9fae16803a63efaa14b87137a793/packages/connectkit/src/components/ConnectKit.tsx#L74).

**Reply by lochie** [reply]
Looks like this one was missed in the [`hideTooltips`](https://docs.family.co/connectkit/api-reference#hideTooltips) option. Would this be suitable or do you think we need a seperate option purely for the SIWE tooltip?

**Reply by Drewsapple** [reply]
Yeah that would be fine for me. I hadn't realized there was already a hideTooltips option, and people with that enabled definitely don't expect the SIWE tooltip to show. 

That makes this more of a bug than a feature request.

I can make a PR that fixes it, but that'll probably be on friday.

**Reply by lochie** [reply]
You're welcome to make a PR, else wise this will be something I'll have time to look into next week. Thanks for opening this discussion, might not have picked this up if it weren't for this!
